include/deal.II/multigrid/mg_constrained_dofs.h

```diff
@@ -100,6 +100,26 @@ class MGConstrainedDoFs : public Subscriptor
     const std::set<types::boundary_id> &boundary_ids,
     const ComponentMask &               component_mask = ComponentMask());
 
+  /**
+   * Fill the internal data structures with information
+   * about no normal flux boundary dofs.
+   *
+   * This function is limited to meshes whose no normal flux boundaries
+   * have faces which are normal to the x-, y-, or z-axis. Also, for a
+   * specific boundary id, all faces must be facing in the same direction,
+   * i.e., a boundary normal to the x-axis must have a different boundary
+   * id than a boundary normal to the y- or z-axis and so on. If the mesh
+   * here was produced using the GridGenerator class, setting
```

Comment:
that's actually a namespace, and there are many functions. Do you want to reference a particular one, say `hyper_cube` or `hyper_rectangle`?